### ISSUE TITLE
ReferenceError: client_id is not defined

### ISSUE THREAD

**Issue by stefarossi** [tykeal/ep_ldapauth#18]
When loading a pad, etherpad crashes with this:
```[2015-01-08 14:24:26.144] [ERROR] console - ReferenceError: client_id is not defined
    at Object.exports.handleMessage [as hook_fn] (/usr/share/etherpad/node_modules/ep_ldapauth/ep_ldapauth.js:178:142)
    at normalizedhook (/usr/share/etherpad/src/static/js/pluginfw/hooks.js:15:27)
    at hookCallWrapper (/usr/share/etherpad/src/static/js/pluginfw/hooks.js:21:14)
    at /usr/share/etherpad/src/static/js/pluginfw/hooks.js:88:7
    at /usr/share/etherpad/src/node_modules/async/lib/async.js:249:17
    at /usr/share/etherpad/src/node_modules/async/lib/async.js:125:13
    at Array.forEach (native)
    at _each (/usr/share/etherpad/src/node_modules/async/lib/async.js:46:24)
    at async.each (/usr/share/etherpad/src/node_modules/async/lib/async.js:124:9)
    at _asyncMap (/usr/share/etherpad/src/node_modules/async/lib/async.js:248:13)```

**Comment by JuRaSSiCBoY** [comment]
Got the same error when loading a pad, did you solve the issue? I think It's because the settings.json variables are not properly configured.

[2015-02-19 13:00:12.004] [ERROR] console - ReferenceError: client_id is not defined
    at Object.exports.handleMessage [as hook_fn] (/var/www/etherpadsave/etherpad/node_modules/ep_ldapauth/ep_ldapauth.js:179:142)
    at normalizedhook (/var/www/etherpadsave/etherpad/src/static/js/pluginfw/hooks.js:15:27)
    at hookCallWrapper (/var/www/etherpadsave/etherpad/src/static/js/pluginfw/hooks.js:21:14)
    at /var/www/etherpadsave/etherpad/src/static/js/pluginfw/hooks.js:88:7
    at /var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:249:17
    at /var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:125:13
    at Array.forEach (native)
    at _each (/var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:46:24)
    at async.each (/var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:124:9)
    at _asyncMap (/var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:248:13)
[2015-02-19 13:00:12.005] [INFO] console - graceful shutdown...
[2015-02-19 13:00:12.082] [INFO] ueberDB - Flushed 1 values
[2015-02-19 13:00:12.187] [INFO] console - db sucessfully closed.

**Comment by Sergio3** [comment]
Got the same error when loading a pad, did you solve the issue? I think It's because the settings.json variables for "ldapauth" are not properly configured.

[2015-02-19 13:00:12.004] [ERROR] console - ReferenceError: client_id is not defined
    at Object.exports.handleMessage [as hook_fn] (/var/www/etherpadsave/etherpad/node_modules/ep_ldapauth/ep_ldapauth.js:179:142)
    at normalizedhook (/var/www/etherpadsave/etherpad/src/static/js/pluginfw/hooks.js:15:27)
    at hookCallWrapper (/var/www/etherpadsave/etherpad/src/static/js/pluginfw/hooks.js:21:14)
    at /var/www/etherpadsave/etherpad/src/static/js/pluginfw/hooks.js:88:7
    at /var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:249:17
    at /var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:125:13
    at Array.forEach (native)
    at _each (/var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:46:24)
    at async.each (/var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:124:9)
    at _asyncMap (/var/www/etherpadsave/etherpad/src/node_modules/async/lib/async.js:248:13)
[2015-02-19 13:00:12.005] [INFO] console - graceful shutdown...
[2015-02-19 13:00:12.082] [INFO] ueberDB - Flushed 1 values
[2015-02-19 13:00:12.187] [INFO] console - db sucessfully closed.


And the the navigator says:

An error occured
The error was reported with the following id: 'lyZHpkQ2XJugAEYw84p7'

Please press and hold Ctrl and press F5 to reload this page, if the problem persists please send this error message to your webmaster:
'ErrorId: lyZHpkQ2XJugAEYw84p7
URL: http://myweb.com:9000/p/proof
UserAgent: Mozilla/5.0 (X11; Linux x86_64; rv:31.0) Gecko/20100101 Firefox/31.0 Iceweasel/31.3.0
TypeError: pad.collabClient is null in http://myweb.com:9000/javascripts/lib/ep_etherpad-lite/static/js/pad.js?callback=require.define at line 254'

**Comment by tykeal** [comment]
This should now be fixed with PR #20 .
